lib/engine/game/g_1860.rb

```diff
@@ -357,7 +357,7 @@ def event_relax_cert_limit!
       end
 
       def event_southern_forms!
-        @log << 'Southern Railway Forms; End of game triggered (via Nationalization).'
+        @log << 'Southern Railway Forms; End Game possible via Nationalisation.'
         @southern_formed = true
```

Comment:
I disagree somewhat - the game is going to end soon. If you want to warn that the game can still end via the bank breaking, you should put that in.

Reply:
I can live with "Southern Railway Form; Nationalization will be triggered when all players’ companies have at least one train"

Reply:
Sure, so something like "Southern Railway Forms; End of game still possible via bank breaking".

If we want to include all end game condtions, maybe something like this: "Southern Railway Forms; End of game possible via Nationalisation, Bank Breaking, or Company hitting max stock value."

And lastly, should I go with nationalisation or nationalization?  The rulebook indicates nationalisation which is the UK spelling, but wasn't sure which to go with for the wording.

Reply:
> I can live with "Southern Railway Forms; Nationalization will be triggered when all players’ companies have at least one train"

Updated! :)